lib/features/recommendation/recommendation_screen.dart

```diff
@@ -11,73 +11,76 @@ class RecommendationScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+        appBar: AppBar(
+          toolbarHeight: 0,
+        ),
         body: Stack(children: [
-      Image.asset(
-        'assets/images/kids_rec_photo.png',
-        fit: BoxFit.cover,
-        height: double.infinity,
-        width: double.infinity,
-        alignment: Alignment.center,
-      ),
-      SizedBox(
-        width: double.infinity,
-        child: BlocBuilder<RecommendationCubit, RecommendationState>(
-          builder: (context, state) {
-            return SafeArea(
-              child: Column(
-                children: [
-                  const SizedBox(height: 20),
-                  Row(
-                    mainAxisAlignment: MainAxisAlignment.end,
+          Image.asset(
+            'assets/images/kids_rec_photo.png',
+            fit: BoxFit.cover,
+            height: double.infinity,
+            width: double.infinity,
+            alignment: Alignment.center,
+          ),
+          SizedBox(
+            width: double.infinity,
+            child: BlocBuilder<RecommendationCubit, RecommendationState>(
+              builder: (context, state) {
+                return SafeArea(
```

Comment:
I think we need to agree either we use SafeArea for all screens or not, since currently we have part with and part of the screens w/o it.
Have you tested it on the ios simulator (SafeArea is about iOS afaik) ?